app/assets/javascripts/components/course_creator/course_creator.jsx

```diff
@@ -418,7 +419,12 @@ const CourseCreator = createReactClass({
         <label htmlFor="checkbox_id">{I18n.t('courses.creator.copy_courses_with_assignments')}</label>
       </span>
     );
-
+    const { ifadmin: ifAdminStr, wiki_ed: wikiEdStr } = document.getElementById('nav_root').dataset;
```

Comment:
I don't think it should be necessary to pull any data from the html for this. I would expect the fix to be centered on the logic for which panels to show and whether to show the newOrClone panel.

Reply:
The issue is with respect to the non-functional back button for instructors